alicloud/cpi.yml

```diff
@@ -0,0 +1,107 @@
+---
+# CPI
+- type: replace
+  path: /releases/-
+  value:
+    name: bosh-alicloud-cpi
+    version: 16
+    url: http://bosh-((region)).oss-((region)).aliyuncs.com/bosh-alicloud-cpi-release-r16.tgz
+    sha1: dc63a7b28fa77294ae4c7305c85d9590ca690cd9
+
+# light stemcell
+- type: replace
+  path: /resource_pools/name=vms/stemcell?
+  value:
+    url: http://bosh-((region)).oss-((region)).aliyuncs.com/light-bosh-stemcell-1016-alicloud-kvm-ubuntu-trusty-go_agent.tgz
+    sha1: ef7dfcf7d677a32625631cdc49257b8680eca9e6
+
+# Configure ECS sizes
+- type: replace
+  path: /resource_pools/name=vms/cloud_properties?
+  value:
+    availability_zone: ((zone))
+    instance_type: "ecs.mn4.small"
+    instance_name: "bosh-director"
```

Comment:
why do we need `instance_name`?

Reply:
It just a name for marking or classifying ECS instance. It is optional.